### ISSUE TITLE
Training Time

### ISSUE THREAD

**Issue by veroveroxie** [nanlliu/Unsupervised-Compositional-Concepts-Discovery#3]
Hi, Thank you so much for your great work!  I really like your work and am trying to run it. I have questions about the code.

1. Do you mind sharing your training time and computing resources for the experiments? I am currently using a single L40 and train my dataset with 100 images. The progress bar shows that it takes about 5 hours to finish. Is it close to your training time?

2.  I found that you optimize the parameters after each backward instead of under accelerator.sync_gradients. Is it intended or a potential bug? In Line 877-880.

`accelerator.backward(loss)
                optimizer.step()
                lr_scheduler.step()
                optimizer.zero_grad()`

3.  After training, how do we generate compositional images, like "a photo of "t1" and "t2""?

Thanks a lot!
